test/end-to-end/pages/screeners/deathcertificate.js

```diff
@@ -1,13 +1,12 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function(answer, testSurvey = false) {
+module.exports = async function(language ='en', answer, testSurvey = false) {
     const I = this;
-
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
     await I.checkPageUrl('app/steps/ui/screeners/deathcertificate');
-    const locator = {css: `#deathCertificate${answer}`};
-    await I.waitForElement(locator);
```

Comment:
removing a wait step like this could have negative impact on the nightly crossbrowser test stability. Relying on URL checks isn't sufficient as Safari's handling of URLs is different to the other browsers